xml/System/Environment.xml

```diff
@@ -1753,8 +1753,8 @@ To identify the operating system platform, for example, Linux or Windows, you ca
         <ReturnType>System.Int32</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets the number of processors on the current machine.</summary>
-        <value>The 32-bit signed integer that specifies the number of processors on the current machine. There is no default. If the current machine contains multiple processor groups, this property returns the number of logical processors that are available for use by the common language runtime (CLR).</value>
+        <summary>Gets the number of processors on the current machine or container (like a Docker container).</summary>
```

Comment:
```suggestion
        <summary>Gets the number of processors available to the current process.</summary>
```
And describe the behavior in detail in remarks. Something like:

```
On Linux and macOS systems in all .NET versions and on Windows systems starting with .NET 6, this API returns the minumum of:
- The physicial number of CPUs on the machine 
- If the process is running with CPU affinity, the number of processors that the process is affinitized to.
- If the process is running with CPU utilization limit, the CPU utilization limit rounded up to the whole number.

The value returned by this API is fixed at .NET runtime startup for the process lifetime. It does not reflect changes in the environment settings while the process is running.
```

Reply:
> - The physicial number of CPUs on the machine

The number of logical processors.  We can leave ourselves some wriggle room by using less technical description, e.g., by saying the returned value reflects CPU affinity and utilization limits enforced by the operating system.